.github/workflows/override-pr-merge-init.yml

```
@@ -0,0 +1,72 @@
+name: Override PR Merge Initialise
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  get-permissions:
+    if: |
+      github.event_name == 'issue_comment' &&
+      github.event.issue.pull_request &&
+      github.event.comment.body == '/merge' &&
+      !github.event.issue.closed_at &&
+      github.event.issue.state == 'open'
+
+    name: Get User Permissions
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    env:
+      USERNAME: ${{ github.event_name == 'issue_comment' && github.event.comment.user.login || github.actor }}

```

Suggest removing this clause as it duplicates a check above.
```suggestion
      USERNAME: ${{ github.event.comment.user.login || github.actor }}

```

---

I have changed this to user just `${{ github.actor }}`.